src/main/kotlin/study/racing/domain/Car.kt

```diff
@@ -1,16 +1,29 @@
 package study.racing.domain
 
-class Car {
+import study.racing.strategy.MoveStrategy
+import study.racing.strategy.RandomMoveStrategy
 
-    private val distance: Distance = Distance()
+class Car(
+    private val strategy: MoveStrategy = RandomMoveStrategy(),
+    name: CarName
+) {
+
+    var distance: Distance = Distance()
+        private set
+    var carName: CarName = name
+        private set
```

Comment:
```suggestion
    val distance: Distance = Distance()
    val carName: CarName = name
```

현재 Car에서는 distance, carName 모두 변경이 되어지지 않는데, 둘 다 모두 불변으로 가도 괜찮지 않을까요?

Reply:
맞습니다. 새로 알려주신 private set이 신기해서 자꾸 적용해보다 보니... 코드 퀄리티가 내려갔네요.
불변으로 변경하겠습니다.